po/LINGUAS

```diff
@@ -8,4 +9,4 @@ oc
 pt_BR
 ru
 ta
-tr
\ No newline at end of file
+tr
```

Comment:
why did this line change?

Reply:
@vixalien I guess adding "es" (in alphabetical order) shifts "tr" one line.

Reply:
no, it's about an added endline at the bottom of the file. no worries though